### PR TITLE
Travis wapiti

### PR DESCRIPTION
**Pull request by whalebot-helmsman**
add commands to install wapiti binary on travis and run test for binary checking

### CODE REVIEW

File: webstruct/wapiti.py [hunk 1]
```diff
@@ -124,6 +124,10 @@ def merge_top_n(chains):
 
 
 class WapitiCRF(BaseSequenceClassifier):
+    """
+    >>> run_command(['which', WapitiCRF.WAPITI_CMD], verbose=False)
```
Comment (kmike) [inline webstruct/wapiti.py]: I think it should be moved to regular tests - WapitiCRF is a public class, its docstring should show its description and usage examples.

File: .travis.yml [hunk 2]
```diff
@@ -29,6 +29,8 @@ addons:
 
 install:
     - pip install -U pip tox codecov
+    - git clone https://github.com/Jekub/Wapiti /tmp/wapiti
+    - pushd /tmp/wapiti && make && sudo make install && popd
```
Comment (kmike) [inline .travis.yml]: 👍 
heh, I wonder why does it work - we're using `sudo: false` on top of travis.yml

**Comment by kmike** [comment]
Thanks @whalebot-helmsman!

**Comment by Kebniss** [comment]
Thank you for this @whalebot-helmsman!

**Pull request merged**